Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
"GET /productos/lista-categorias/ HTTP/1.1" 200 19875
"GET /static/assets/css/categorias.css HTTP/1.1" 200 782
"GET /productos/lista-marcas/ HTTP/1.1" 200 19466
"GET /static/assets/css/marcas.css HTTP/1.1" 200 776
"GET /productos/lista-proveedores/ HTTP/1.1" 200 18071
"GET /static/assets/css/proveedores.css HTTP/1.1" 200 776
"GET /contacto/ HTTP/1.1" 200 16421
"GET /usuario/lista-usuarios/ HTTP/1.1" 200 19933
Internal Server Error: /usuario/editar-usuario/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 890, in _resolve_lookup
    raise TypeError
TypeError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py", line 242, in editar_perfil
    return render(request, 'usuarios/editar_perfil.html', context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1075, in render
    output = self.filter_expression.resolve(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 722, in resolve
    obj = self.var.resolve(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 854, in resolve
    value = self._resolve_lookup(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 901, in _resolve_lookup
    current = getattr(current, bit)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\files.py", line 69, in url
    self._require_file()
    ~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\files.py", line 44, in _require_file
    raise ValueError(
        "The '%s' attribute has no file associated with it." % self.field.name
    )
ValueError: The 'foto_perfil' attribute has no file associated with it.
"GET /usuario/editar-usuario/ HTTP/1.1" 500 191762
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 65382)
"GET /usuario/lista-usuarios/ HTTP/1.1" 200 19931
Internal Server Error: /usuario/editar-usuario/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 890, in _resolve_lookup
    raise TypeError
TypeError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py", line 242, in editar_perfil
    return render(request, 'usuarios/editar_perfil.html', context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1075, in render
    output = self.filter_expression.resolve(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 722, in resolve
    obj = self.var.resolve(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 854, in resolve
    value = self._resolve_lookup(context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 901, in _resolve_lookup
    current = getattr(current, bit)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\files.py", line 69, in url
    self._require_file()
    ~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\files.py", line 44, in _require_file
    raise ValueError(
        "The '%s' attribute has no file associated with it." % self.field.name
    )
ValueError: The 'foto_perfil' attribute has no file associated with it.
"GET /usuario/editar-usuario/ HTTP/1.1" 500 191762
"GET /usuario/editar-usuario/ HTTP/1.1" 200 16738
- Broken pipe from ('127.0.0.1', 65253)
"GET /usuario/editar-usuario/ HTTP/1.1" 200 16738
- Broken pipe from ('127.0.0.1', 51910)
"GET /contacto/ HTTP/1.1" 200 16420
"GET /productos/lista-proveedores/ HTTP/1.1" 200 18071
"GET /productos/lista-marcas/ HTTP/1.1" 200 19466
"GET /productos/lista/ HTTP/1.1" 200 18499
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /productos/crear-producto/ HTTP/1.1" 200 18890
"POST /productos/crear-producto/ HTTP/1.1" 302 0
"GET /productos/lista-productos/ HTTP/1.1" 200 21830
"GET /media/imagenes/L5590-690x460-1.webp HTTP/1.1" 200 11962
"GET /media/imagenes/images_O7jCtbm.jpg HTTP/1.1" 200 7756
"GET /media/imagenes/Mesa_de_trabajo_50_1_ncKmXpz.webp HTTP/1.1" 200 1464940
"GET /media/imagenes/Mesa_de_trabajo_50_1_BI2qgia.webp HTTP/1.1" 200 1464940
"GET /__debug__/render_panel/?store_id=143a168eb5ee44a0bce91ae0b549ab10&panel_id=CachePanel HTTP/1.1" 200 1441
"GET /productos/lista/ HTTP/1.1" 200 18497
"GET /productos/lista-categorias/ HTTP/1.1" 200 19874
"GET /productos/lista-categorias/ HTTP/1.1" 200 19874
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/assets/css/categorias.css HTTP/1.1" 200 782
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /__debug__/render_panel/?store_id=c38a24b2b58c467388bf89faff0d7540&panel_id=SQLPanel HTTP/1.1" 200 41357
"GET /__debug__/render_panel/?store_id=c38a24b2b58c467388bf89faff0d7540&panel_id=StaticFilesPanel HTTP/1.1" 200 57392
"GET /productos/lista/ HTTP/1.1" 200 18499
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /productos/lista/ HTTP/1.1" 200 18496
"GET /productos/lista/ HTTP/1.1" 200 18496
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /productos/lista-categorias/ HTTP/1.1" 200 19874
"GET /productos/lista/ HTTP/1.1" 200 18496
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /productos/lista/ HTTP/1.1" 200 18500
- Broken pipe from ('127.0.0.1', 63367)
- Broken pipe from ('127.0.0.1', 63447)
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /productos/lista/ HTTP/1.1" 200 18500
"GET /productos/lista/ HTTP/1.1" 200 18499
"GET /static/assets/js/base.js HTTP/1.1" 200 602
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /productos/lista/ HTTP/1.1" 302 0
"GET /es/productos/lista/ HTTP/1.1" 200 18554
"GET /en/productos/lista/ HTTP/1.1" 200 18171
"GET /en/productos/lista/ HTTP/1.1" 200 18171
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 19583
"GET /en/productos/lista/ HTTP/1.1" 200 18172
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 19582
"GET /en/productos/lista/ HTTP/1.1" 200 18171
"GET /en/ HTTP/1.1" 200 15472
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 302 0
"GET /es/admin/login/?next=/es/admin/ HTTP/1.1" 200 17767
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
"GET /static/admin/js/theme.js HTTP/1.1" 304 0
"GET /static/admin/css/base.css HTTP/1.1" 200 22120
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/admin/css/login.css HTTP/1.1" 200 951
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
"GET /static/admin/css/responsive.css HTTP/1.1" 200 16565
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"POST /es/admin/login/?next=/es/admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 200 33397
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
"GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
Internal Server Error: /es/admin/productos/producto/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: productos_proveedor.nombre_es

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 867, in get_changelist_instance
    return ChangeList(
        request,
    ...<12 lines>...
        self.search_help_text,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 545, in get_queryset
    ) = self.get_filters(request)
        ~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 216, in get_filters
    spec = field_list_filter_class(
        field,
    ...<4 lines>...
        field_path=field_path,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 217, in create
    return list_filter_class(
        field, request, params, model, model_admin, field_path=field_path
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 232, in __init__
    self.lookup_choices = self.field_choices(field, request, model_admin)
                          ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 273, in field_choices
    return field.get_choices(include_blank=False, ordering=ordering)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\__init__.py", line 1083, in get_choices
    (choice_func(x), str(x)) for x in qs
                                      ^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 384, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: productos_proveedor.nombre_es
"GET /es/admin/productos/producto/ HTTP/1.1" 500 250274
Internal Server Error: /es/admin/productos/producto/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: productos_proveedor.nombre_es

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 867, in get_changelist_instance
    return ChangeList(
        request,
    ...<12 lines>...
        self.search_help_text,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 545, in get_queryset
    ) = self.get_filters(request)
        ~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 216, in get_filters
    spec = field_list_filter_class(
        field,
    ...<4 lines>...
        field_path=field_path,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 217, in create
    return list_filter_class(
        field, request, params, model, model_admin, field_path=field_path
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 232, in __init__
    self.lookup_choices = self.field_choices(field, request, model_admin)
                          ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 273, in field_choices
    return field.get_choices(include_blank=False, ordering=ordering)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\__init__.py", line 1083, in get_choices
    (choice_func(x), str(x)) for x in qs
                                      ^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 384, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: productos_proveedor.nombre_es
"GET /es/admin/productos/producto/ HTTP/1.1" 500 250537
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
Not Found: /favicon.ico
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /favicon.ico HTTP/1.1" 404 16317
Watching for file changes with StatReloader
Internal Server Error: /es/admin/productos/producto/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: productos_proveedor.nombre_es

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 867, in get_changelist_instance
    return ChangeList(
        request,
    ...<12 lines>...
        self.search_help_text,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 545, in get_queryset
    ) = self.get_filters(request)
        ~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 216, in get_filters
    spec = field_list_filter_class(
        field,
    ...<4 lines>...
        field_path=field_path,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 217, in create
    return list_filter_class(
        field, request, params, model, model_admin, field_path=field_path
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 232, in __init__
    self.lookup_choices = self.field_choices(field, request, model_admin)
                          ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 273, in field_choices
    return field.get_choices(include_blank=False, ordering=ordering)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\__init__.py", line 1083, in get_choices
    (choice_func(x), str(x)) for x in qs
                                      ^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 384, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: productos_proveedor.nombre_es
"GET /es/admin/productos/producto/ HTTP/1.1" 500 250538
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"GET /es/admin/ HTTP/1.1" 200 33397
Internal Server Error: /es/admin/productos/proveedor/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: productos_proveedor.nombre_es

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2163, in changelist_view
    "selection_note": _("0 of %(cnt)s selected") % {"cnt": len(cl.result_list)},
                                                           ~~~^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 366, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: productos_proveedor.nombre_es
"GET /es/admin/productos/proveedor/ HTTP/1.1" 500 208806
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 62887)
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\translation.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\translation.py changed, reloading.
Watching for file changes with StatReloader
"GET /es/admin/productos/proveedor/ HTTP/1.1" 200 31480
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/js/core.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
- Broken pipe from ('127.0.0.1', 55099)
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/js/filters.js HTTP/1.1" 200 978
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 55095)
"GET /es/admin/productos/proveedor/5/change/ HTTP/1.1" 200 30655
"GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
"GET /static/admin/js/change_form.js HTTP/1.1" 200 606
"GET /static/admin/css/forms.css HTTP/1.1" 200 8525
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/css/widgets.css HTTP/1.1" 200 11991
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 55096)
"GET /es/admin/productos/producto/ HTTP/1.1" 200 38054
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/img/search.svg HTTP/1.1" 200 458
"GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
"GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
"GET /es/admin/productos/producto/2/change/ HTTP/1.1" 200 42035
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 55100)
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32175
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/categoria/14/change/ HTTP/1.1" 200 34425
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Watching for file changes with StatReloader
"POST /es/admin/productos/categoria/14/change/ HTTP/1.1" 200 34660
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/14/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/14/change/ HTTP/1.1" 200 34796
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32184
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 49761)
"GET /es/admin/productos/categoria/10/change/ HTTP/1.1" 200 34425
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 50798)
"POST /es/admin/productos/categoria/10/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/10/change/ HTTP/1.1" 200 34814
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 50972)
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32194
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/categoria/9/change/ HTTP/1.1" 200 34421
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/9/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32414
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 51210)
"GET /es/admin/productos/categoria/6/change/ HTTP/1.1" 200 34421
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/6/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32431
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 51592)
"GET /es/admin/productos/categoria/4/change/ HTTP/1.1" 200 34422
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/4/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32441
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 51869)
"GET /es/admin/productos/categoria/3/change/ HTTP/1.1" 200 34421
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/3/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32467
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 51870)
"GET /es/admin/productos/categoria/2/change/ HTTP/1.1" 200 34421
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/categoria/2/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32484
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 52135)
"GET /es/admin/productos/categoria/1/change/ HTTP/1.1" 200 34422
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 52340)
"POST /es/admin/productos/categoria/1/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/categoria/ HTTP/1.1" 200 32489
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 52470)
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\models.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 302 0
"GET /es/ HTTP/1.1" 200 15852
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/assets/css/base.css HTTP/1.1" 304 0
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /es/ HTTP/1.1" 200 15850
"GET /es/ HTTP/1.1" 200 15850
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
Internal Server Error: /es/logout/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py", line 30, in logout_view
    auth_logout(request)
    ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\auth\__init__.py", line 254, in logout
    request.session.flush()
    ~~~~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\sessions\backends\base.py", line 418, in flush
    self.delete()
    ~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\sessions\backends\db.py", line 175, in delete
    self.model.objects.get(session_key=session_key).delete()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\base.py", line 1281, in delete
    return collector.delete()
           ~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\deletion.py", line 448, in delete
    count = sql.DeleteQuery(model).delete_batch(
        [instance.pk], self.using
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\subqueries.py", line 42, in delete_batch
    num_deleted += self.do_query(
                   ~~~~~~~~~~~~~^
        self.get_meta().db_table, self.where, using=using
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\subqueries.py", line 24, in do_query
    return self.get_compiler(using).execute_sql(ROW_COUNT)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: database is locked
- Broken pipe from ('127.0.0.1', 53857)
"GET /es/productos/lista/ HTTP/1.1" 200 18503
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /es/admin/productos/producto/ HTTP/1.1" 200 38110
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/2/change/ HTTP/1.1" 200 42091
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Internal Server Error: /es/admin/productos/producto/2/change/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\base\base.py", line 303, in _commit
    return self.connection.commit()
           ~~~~~~~~~~~~~~~~~~~~~~^^
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 1987, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 1842, in changeform_view
    with transaction.atomic(using=router.db_for_write(self.model)):
         ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\transaction.py", line 263, in __exit__
    connection.commit()
    ~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\base\base.py", line 327, in commit
    self._commit()
    ~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\base\base.py", line 302, in _commit
    with debug_transaction(self, "COMMIT"), self.wrap_database_errors:
                                            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\base\base.py", line 303, in _commit
    return self.connection.commit()
           ~~~~~~~~~~~~~~~~~~~~~~^^
django.db.utils.OperationalError: database is locked
"POST /es/admin/productos/producto/2/change/ HTTP/1.1" 500 160582
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"POST /es/admin/productos/producto/2/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/producto/2/change/ HTTP/1.1" 200 42685
- Broken pipe from ('127.0.0.1', 54131)
- Broken pipe from ('127.0.0.1', 54127)
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"GET /es/admin/productos/producto/ HTTP/1.1" 200 38133
- Broken pipe from ('127.0.0.1', 54128)
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/10/change/ HTTP/1.1" 200 42093
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/10/change/ HTTP/1.1" 302 0
"GET /es/admin/productos/producto/ HTTP/1.1" 200 38389
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39089
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/3/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42210
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/3/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39336
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/8/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42217
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/8/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39360
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/11/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42224
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/11/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39389
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/1/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42231
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/1/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39427
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/12/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42210
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/12/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/?o=1.-3 HTTP/1.1" 200 39450
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /es/admin/productos/producto/13/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42199
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"POST /es/admin/productos/producto/13/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 302 0
"GET /es/admin/productos/producto/13/change/?_changelist_filters=o%3D1.-3 HTTP/1.1" 200 42551
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 54130)
"GET /es/productos/lista/ HTTP/1.1" 200 18500
"GET /es/productos/lista/ HTTP/1.1" 200 18500
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /en/productos/lista/ HTTP/1.1" 200 18122
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /en/productos/lista/ HTTP/1.1" 200 18804
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18804
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
- Broken pipe from ('127.0.0.1', 64487)
- Broken pipe from ('127.0.0.1', 64486)
- Broken pipe from ('127.0.0.1', 64483)
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
- Broken pipe from ('127.0.0.1', 64502)
- Broken pipe from ('127.0.0.1', 64499)
- Broken pipe from ('127.0.0.1', 64498)
"GET /en/productos/lista/ HTTP/1.1" 200 18760
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
- Broken pipe from ('127.0.0.1', 64505)
- Broken pipe from ('127.0.0.1', 64515)
- Broken pipe from ('127.0.0.1', 64516)
- Broken pipe from ('127.0.0.1', 64517)
- Broken pipe from ('127.0.0.1', 64518)
- Broken pipe from ('127.0.0.1', 64519)
- Broken pipe from ('127.0.0.1', 64520)
- Broken pipe from ('127.0.0.1', 64525)
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /en/productos/lista/ HTTP/1.1" 200 18759
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 20218
"GET /static/assets/css/categorias.css HTTP/1.1" 200 782
"GET /fr/productos/lista-categorias/ HTTP/1.1" 200 20427
"GET /es/productos/lista-categorias/ HTTP/1.1" 200 20603
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 20216
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /fr/productos/lista-categorias/ HTTP/1.1" 200 20427
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /es/productos/lista-categorias/ HTTP/1.1" 200 20603
"GET /es/productos/lista/ HTTP/1.1" 200 19138
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /es/productos/lista/ HTTP/1.1" 200 19200
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /fr/productos/lista/ HTTP/1.1" 200 18967
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /es/productos/lista/ HTTP/1.1" 200 19194
Watching for file changes with StatReloader
"GET /es/productos/lista/ HTTP/1.1" 200 19199
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18811
Watching for file changes with StatReloader
"GET /en/productos/lista/ HTTP/1.1" 200 18816
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /es/productos/lista/ HTTP/1.1" 200 19194
"GET /es/productos/lista/ HTTP/1.1" 200 19194
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /es/productos/lista/ HTTP/1.1" 200 19194
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
- Broken pipe from ('127.0.0.1', 57065)
"GET /es/productos/lista/ HTTP/1.1" 200 19194
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /es/productos/lista/ HTTP/1.1" 200 19195
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /en/productos/lista/ HTTP/1.1" 200 18811
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /en/productos/actualizar-producto/1 HTTP/1.1" 200 20380
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 15951
"GET /en/productos/crear-producto/ HTTP/1.1" 200 20178
Internal Server Error: /en/productos/lista-categorias/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
    ...<8 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
"GET /en/productos/lista-categorias/ HTTP/1.1" 500 233197
Internal Server Error: /en/productos/lista-categorias/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
    ...<8 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'trans', expected 'endblock'. Did you forget to register or load this tag?
"GET /en/productos/lista-categorias/ HTTP/1.1" 500 229928
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 15951
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
Internal Server Error: /en/productos/lista-categorias/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
    ...<8 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'trans', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
"GET /en/productos/lista-categorias/ HTTP/1.1" 500 277119
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 15951
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 20117
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/assets/css/categorias.css HTTP/1.1" 200 782
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 20114
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/assets/css/categorias.css HTTP/1.1" 200 782
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /es/productos/lista-categorias/ HTTP/1.1" 200 20541
"POST /i18n/setlang/ HTTP/1.1" 302 0
"GET /en/productos/lista-categorias/ HTTP/1.1" 200 20113
Watching for file changes with StatReloader
Not Found: /productoszs
"GET /productoszs HTTP/1.1" 404 16318
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
Not Found: /es/productoszs
"GET /es/productoszs HTTP/1.1" 404 19559
Not Found: /es/ productos/api/productos
"GET /es/%20productos/api/productos HTTP/1.1" 404 19615
"GET /es/productos/api/productos HTTP/1.1" 301 0
"GET /es/productos/api/productos/ HTTP/1.1" 200 33118
"GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3426
"GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
"GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
"GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
"GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1793
"GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
"GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
"GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
"GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 200 59
"GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3796
"GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 200 87533
"GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
"GET /es/productos/api/productos/ HTTP/1.1" 200 33116
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
Watching for file changes with StatReloader
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\urls.py changed, reloading.
Watching for file changes with StatReloader
"POST /usuario/api/registro HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"GET /admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 302 0
"GET /es/admin/login/?next=/es/admin/ HTTP/1.1" 200 17767
"GET /static/admin/css/base.css HTTP/1.1" 200 22120
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/admin/js/theme.js HTTP/1.1" 304 0
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
"GET /static/admin/css/login.css HTTP/1.1" 200 951
"GET /static/admin/css/responsive.css HTTP/1.1" 200 16565
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"POST /es/admin/login/?next=/es/admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 200 33573
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"GET /es/admin/auth/user/ HTTP/1.1" 200 36107
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 304 0
"GET /static/admin/js/core.js HTTP/1.1" 304 0
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
- Broken pipe from ('127.0.0.1', 55714)
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
"GET /static/admin/img/search.svg HTTP/1.1" 200 458
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
"GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
Watching for file changes with StatReloader
"POST /usuario/api/registro HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /usuario/api/registro HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"GET /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 19241
"GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
"GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
"GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3796
"GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
"GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1793
"GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3426
"GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 200 59
"GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
"GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
"GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
"GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
"GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 200 87533
- Broken pipe from ('127.0.0.1', 63299)
"POST /es/usuario/api/registro/ HTTP/1.1" 201 19479
"GET /__debug__/history_sidebar/?store_id=9558b30554814141809cf2e211e3de19 HTTP/1.1" 200 10301
"GET /es/admin/auth/user/ HTTP/1.1" 200 36580
- Broken pipe from ('127.0.0.1', 63296)
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 63300)
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Method Not Allowed: /es/usuario/api/registro/
"GET /es/usuario/api/registro/ HTTP/1.1" 405 42
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
"GET /es/productos/api/productos/ HTTP/1.1" 200 3997
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
"OPTIONS /usuario/api/registro/ HTTP/1.1" 200 0
"POST /usuario/api/registro/ HTTP/1.1" 302 0
Bad Request: /es/usuario/api/registro/
"POST /es/usuario/api/registro/ HTTP/1.1" 400 135
"OPTIONS /es/usuario/api/registro/ HTTP/1.1" 200 0
"POST /es/usuario/api/registro/ HTTP/1.1" 201 120
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\urls.py changed, reloading.
Watching for file changes with StatReloader
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"OPTIONS /es/usuario/api/login/ HTTP/1.1" 200 0
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"OPTIONS /es/usuario/api/login/ HTTP/1.1" 200 0
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\urls.py changed, reloading.
Watching for file changes with StatReloader
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 184
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 133
"OPTIONS /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 0
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
Unauthorized: /es/usuario/api/login/
"POST /es/usuario/api/login/ HTTP/1.1" 401 76
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\usuario\views.py changed, reloading.
Watching for file changes with StatReloader
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"OPTIONS /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 0
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"OPTIONS /es/usuario/api/login/ HTTP/1.1" 200 0
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
Unauthorized: /es/usuario/api/validar-autenticacion/
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 401 65
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 302 0
"GET /es/admin/login/?next=/es/admin/ HTTP/1.1" 200 17767
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
"GET /static/admin/css/base.css HTTP/1.1" 200 22120
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/admin/css/responsive.css HTTP/1.1" 200 16565
"GET /static/admin/js/theme.js HTTP/1.1" 304 0
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
"GET /static/admin/css/login.css HTTP/1.1" 200 951
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"POST /es/admin/login/?next=/es/admin/ HTTP/1.1" 302 0
"GET /es/admin/ HTTP/1.1" 200 29794
"GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
"GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
"GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16318
"GET /es/admin/auth/user/ HTTP/1.1" 200 32953
"GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
"GET /static/admin/js/urlify.js HTTP/1.1" 304 0
"GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
"GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
"GET /static/admin/js/core.js HTTP/1.1" 304 0
"GET /static/admin/js/actions.js HTTP/1.1" 304 0
"GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
"GET /static/admin/js/filters.js HTTP/1.1" 200 978
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
"GET /static/admin/img/search.svg HTTP/1.1" 200 458
"GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
"GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
"GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
"GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
"GET /es/admin/ HTTP/1.1" 200 29794
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 61896)
Internal Server Error: /es/admin/productos/producto/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
psycopg2.errors.UndefinedColumn: no existe la columna productos_categoria.descripcion
LINE 1: ..."."nombre_en", "productos_categoria"."nombre_fr", "productos...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2024, in changelist_view
    cl = self.get_changelist_instance(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 867, in get_changelist_instance
    return ChangeList(
        request,
    ...<12 lines>...
        self.search_help_text,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 545, in get_queryset
    ) = self.get_filters(request)
        ~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\views\main.py", line 216, in get_filters
    spec = field_list_filter_class(
        field,
    ...<4 lines>...
        field_path=field_path,
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 217, in create
    return list_filter_class(
        field, request, params, model, model_admin, field_path=field_path
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 232, in __init__
    self.lookup_choices = self.field_choices(field, request, model_admin)
                          ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\filters.py", line 273, in field_choices
    return field.get_choices(include_blank=False, ordering=ordering)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\fields\__init__.py", line 1083, in get_choices
    (choice_func(x), str(x)) for x in qs
                                      ^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 384, in __iter__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
django.db.utils.ProgrammingError: no existe la columna productos_categoria.descripcion
LINE 1: ..."."nombre_en", "productos_categoria"."nombre_fr", "productos...
                                                             ^

"GET /es/admin/productos/producto/ HTTP/1.1" 500 243222
"GET /es/admin/ HTTP/1.1" 200 29794
Internal Server Error: /es/admin/productos/categoria/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
psycopg2.errors.UndefinedColumn: no existe la columna productos_categoria.descripcion
LINE 1: ..."."nombre_en", "productos_categoria"."nombre_fr", "productos...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\contrib\admin\options.py", line 2163, in changelist_view
    "selection_note": _("0 of %(cnt)s selected") % {"cnt": len(cl.result_list)},
                                                           ~~~^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 366, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\models\sql\compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 235, in execute
    return self._record(super().execute, sql, params)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\debug_toolbar\panels\sql\tracking.py", line 160, in _record
    return method(sql, params)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
django.db.utils.ProgrammingError: no existe la columna productos_categoria.descripcion
LINE 1: ..."."nombre_en", "productos_categoria"."nombre_fr", "productos...
                                                             ^

"GET /es/admin/productos/categoria/ HTTP/1.1" 500 202088
"GET /es/admin/ HTTP/1.1" 200 29794
"GET /es/admin/usuario/perfilusuario/ HTTP/1.1" 200 30376
"GET /es/admin/jsi18n/ HTTP/1.1" 200 8691
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 61895)
"GET /es/admin/ HTTP/1.1" 200 29794
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\tienda\settings.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda\productos\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 302 0
"GET /es/ HTTP/1.1" 200 16371
"GET /static/assets/css/base.css HTTP/1.1" 200 3829
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/bootstrap/js/bootstrap.bundle.min.js HTTP/1.1" 200 80821
"GET /static/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 232018
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET / HTTP/1.1" 302 0
"GET /es/ HTTP/1.1" 200 16371
- Broken pipe from ('127.0.0.1', 62621)
- Broken pipe from ('127.0.0.1', 62623)
"OPTIONS /es/usuario/api/login/ HTTP/1.1" 200 0
"POST /es/usuario/api/login/ HTTP/1.1" 200 570
"OPTIONS /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 0
"OPTIONS /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 0
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
"GET /es/usuario/api/validar-autenticacion/ HTTP/1.1" 200 131
Watching for file changes with StatReloader
"GET /es/ HTTP/1.1" 200 16372
- Broken pipe from ('127.0.0.1', 49644)
Watching for file changes with StatReloader
"GET /es/ HTTP/1.1" 200 16371
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda-html\productos\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda-html\productos\urls.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda-html\productos\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /es/produtos
"GET /es/produtos HTTP/1.1" 404 19547
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
Not Found: /es/ productos/ productos
"GET /es/%20productos/%20productos HTTP/1.1" 404 19607
"GET /es/productos/productos HTTP/1.1" 301 0
"GET /es/productos/productos/ HTTP/1.1" 302 0
"GET /es/?next=/es/productos/productos/ HTTP/1.1" 200 16401
"GET /es/login/ HTTP/1.1" 200 16721
"POST /es/login/ HTTP/1.1" 302 0
"GET /es/ HTTP/1.1" 200 16498
"GET /es/productos/productos/ HTTP/1.1" 200 19221
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /media/imagenes/56a38571dde36168af1b9ba61d441b89.webp HTTP/1.1" 200 42558
"GET /media/imagenes/photo-1560185128-e173042f79dd.jpg HTTP/1.1" 200 784473
"GET /media/imagenes/photo-1607870411590-d5e9e06da09a.jpg HTTP/1.1" 200 521744
"GET /es/productos/productos/ HTTP/1.1" 200 19217
"GET /es/productos/productos/ HTTP/1.1" 200 19217
"GET /es/productos/productos/ HTTP/1.1" 200 18404
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /es/productos/productos/ HTTP/1.1" 200 20112
"GET /static/assets/js/main.js HTTP/1.1" 200 9040
"GET /es/productos/productos/ HTTP/1.1" 302 0
"GET /es/?next=/es/productos/productos/ HTTP/1.1" 200 16401
"GET /es/productos/productos/ HTTP/1.1" 200 20112
- Broken pipe from ('127.0.0.1', 62515)
"GET /es/productos/productos/ HTTP/1.1" 200 20112
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/assets/js/base.js HTTP/1.1" 200 602
"GET /static/assets/js/main.js HTTP/1.1" 200 9040
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
- Broken pipe from ('127.0.0.1', 64888)
- Broken pipe from ('127.0.0.1', 64891)
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29758
"GET /es/productos/productos/ HTTP/1.1" 200 20112
- Broken pipe from ('127.0.0.1', 64890)
- Broken pipe from ('127.0.0.1', 64892)
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29758
"GET /es/productos/productos/ HTTP/1.1" 200 20112
- Broken pipe from ('127.0.0.1', 64889)
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29757
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 16317
"GET /es/productos/productos/ HTTP/1.1" 200 20063
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /static/assets/css/productos.css HTTP/1.1" 200 776
"GET /static/assets/js/main.js HTTP/1.1" 200 9040
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29757
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
Not Found: /es/productos/productos/categories.html
"GET /es/productos/productos/categories.html HTTP/1.1" 404 29746
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29757
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 49661)
Not Found: /es/productos/productos/installHook.js.map
"GET /es/productos/productos/installHook.js.map HTTP/1.1" 404 29757
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda-html\productos\views.py changed, reloading.
Watching for file changes with StatReloader
D:\Clases\Python\Enyoi\Ejercicios\Python\tienda-html\productos\urls.py changed, reloading.
Watching for file changes with StatReloader
"GET /es/productos/categorias-html/ HTTP/1.1" 200 15669
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
"GET /es/productos/categorias-html/ HTTP/1.1" 200 15667
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
- Broken pipe from ('127.0.0.1', 51942)
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /es/productos/categorias-html/ HTTP/1.1" 200 15667
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /es/productos/categorias-html/ HTTP/1.1" 200 20724
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 14803
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4692
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
Not Found: /es/productos/categorias-html/products.html
"GET /es/productos/categorias-html/products.html HTTP/1.1" 404 30137
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
Not Found: /favicon.ico
- Broken pipe from ('127.0.0.1', 56503)
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
Internal Server Error: /es/productos/categorias-html/
Traceback (most recent call last):
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\test\utils.py", line 114, in instrumented_test_render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\urls\base.py", line 98, in reverse
    resolved_url = resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Clases\Python\Enyoi\Ejercicios\Python\venv_tienda\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'list_productos_html' not found. 'list_productos_html' is not a valid view function or pattern name.
"GET /es/productos/categorias-html/ HTTP/1.1" 500 152919
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30158
"GET /es/productos/categorias-html/ HTTP/1.1" 200 20749
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
"GET /es/productos/categorias-html/ HTTP/1.1" 200 20749
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30158
"GET /static/assets/css/styles.css HTTP/1.1" 200 1135
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 29444
Not Found: /es/productos/categorias-html/index.html
"GET /es/productos/categorias-html/index.html HTTP/1.1" 404 30126
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
Not Found: /es/productos/categorias-html/installHook.js.map
"GET /es/productos/categorias-html/installHook.js.map HTTP/1.1" 404 30157
- Broken pipe from ('127.0.0.1', 56502)
- Broken pipe from ('127.0.0.1', 58373)
- Broken pipe from ('127.0.0.1', 56393)
- Broken pipe from ('127.0.0.1', 58374)
"GET / HTTP/1.1" 302 0
"GET /es/ HTTP/1.1" 200 16498
"GET /es/productos/productos/ HTTP/1.1" 200 20092
"GET /es/productos/categorias-html/ HTTP/1.1" 200 19536
"GET /es/productos/categorias-html/ HTTP/1.1" 200 360539
